setup.py

```diff
@@ -24,5 +24,5 @@
         "Topic :: Software Development",
         "Topic :: Software Development :: Version Control"
     ],
-    install_requires=["github3.py>=0.9.0,<1.0","whelk>=2.6", "docopt>=0.5.0", "six"],
+    install_requires=["github3.py>=1.0.0,<=1.1", "whelk>=2.6", "docopt>=0.5.0", "six"],
```

Comment:
I actually want to use 1.1.0. No reason to do the big change and not take the latest version :smile:

Reply:
done